### ISSUE TITLE
Code is unsafe and the same DLL file is used regardless of the target OS/arch

### ISSUE THREAD

**Issue by smaybius** [Starpelly/MiniaudioSharp#1]
Because unsafe mode is on in order for the pointer logic to be ported as-is, it expects programs implementing it to enable unsafe code as well. The miniaudio.dll doesn't change if you're meaning to use miniaudio on a different OS or Windows on ARM. BASS, a proprietary similarity to miniaudio, has different ports for each OS and arch. On the latter of the titled issue though, there should either be a centralized hub for all ports of miniaudio, or there should be a script to compile miniaudio.h to the targeted OS and arch.

**Comment by Starpelly** [comment]
Yeah sorry, I made this for a project I dropped a while ago and I guess I forgot to deprecate it.

I switched to Beef recently so this library is useless to me now. If you want it I can transfer it to you, or I can just update with proper build instructions.
